### ISSUE TITLE
Ajustar dependência abnt-ibid.cbx

### ISSUE THREAD

**Issue by admwagner** [abntex/biblatex-abnt#58]
O pacote l3regex somente é encontrado no Ubuntu até a versão "bionic". 
O compilador dá aviso para trocar pelo pacote "expl3"

Fiz o commit da alteração, mas seria bom mais alguém testar.

**Comment by moewew** [comment]
Sorry, I don't speak Portuguese, so English will have to be enough.

This was reported in https://github.com/abntex/biblatex-abnt/issues/53 and fixed in `dev` with https://github.com/abntex/biblatex-abnt/pull/54, but apparently the `dev` branch hasn't been merged into `master` for a while. Maybe it is time for a merge and a new release at some point?

**Comment by admwagner** [comment]
Sim. Desculpem, não havia procurado nas tarefas fechadas. Terei mais cuidado.

**Comment by dbmrq** [comment]
> This was reported in #53 and fixed in `dev` with #54, but apparently the `dev` branch hasn't been merged into `master` for a while. Maybe it is time for a merge and a new release at some point?

Yes, the only reason these changes haven’t been merged yet is that there have been changes to the ABNT rules and the dev branch only implements some of them, so it’s a mix of the old and new rules. I was hoping to merge when I could check what else needs to be changed, but it’s been hard to find the time, so I think it might be best to make a provisional release, even if the new norms aren’t fully taken into account. I’ll do that as soon as I can. Thank you for keeping an eye out for our humble and poorly maintained style. 😅
